src/components/CellViewer/index.tsx

```diff
@@ -19,19 +19,21 @@ const CellViewer: React.FunctionComponent<VolumeViewerProps> = (props) => {
 
     return (
         <div className={styles.cellViewerContainer} style={CONTAINER_STYLE}>
-            <ImageViewerApp
-                cellId={props.cellId}
-                imageUrl={props.baseUrl + props.cellPath}
-                parentImageDownloadHref={props.fovDownloadHref}
-                imageDownloadHref={props.cellDownloadHref}
-                parentImageUrl={props.baseUrl + props.fovPath}
-                viewerChannelSettings={props.viewerChannelSettings}
-                transform={props.transform}
-                onControlPanelToggle={props.onControlPanelToggle}
-                metadata={props.metadata}
-                appHeight="100%"
-                canvasMargin="0 0 0 0"
-            />
+            <ViewerStateProvider>
+                <ImageViewerApp
+                    cellId={props.cellId}
+                    imageUrl={props.baseUrl + props.cellPath}
+                    imageDownloadHref={props.cellDownloadHref}
+                    parentImageUrl={props.fovPath ? props.baseUrl + props.fovPath : ""}
```

Comment:
This is easier to see with whitespace disabled, but all I did was change this to return an empty string if `props.fovPath` is an empty string.